lib/releaselib.sh

```diff
@@ -966,7 +965,7 @@ release::docker::release_from_tarfiles () {
   local registry=$1
   local version=$2
   local build_output=$3
-  local docker_push_cmd=(docker)
+  local docker_push_cmd=("$GCLOUD" "docker" "--")
```

Comment:
This should always be safe and work regardless of gcloud version:
`local docker_push_cmd=("$GCLOUD" "docker" "-s" "$registry" "--")`